Example/Core/Tests/FIRAppAssociationRegistrationUnitTests.m

```diff
@@ -52,8 +50,8 @@ - (void)testPassObject {
   id result = [FIRAppAssociationRegistration registeredObjectWithHost:host
                                                                   key:kKey
                                                         creationBlock:^id _Nullable() {
-    return obj;
-  }];
+                                                          return obj;
```

Comment:
It should be OK in the short term to have different style guidelines by component.  We can revisit if we get a higher sustained volume of external PR's with style issues.

Maybe we should fix clang-format?